config.go

```diff
@@ -936,15 +939,15 @@ func loadConfig() (*config, error) {
 
 	// Finally, ensure that we are only listening on localhost if Tor
 	// inbound support is enabled.
-	if cfg.Tor.V2 || cfg.Tor.V3 {
+	if (cfg.Tor.V2 || cfg.Tor.V3) && !(len(cfg.Tor.ControlPassword) > 0) {
```

Comment:
What's the rationale for this change? We can't assume that if a ControlPassword is provided then we're dealing with a remote Tor server.

Reply:
Right, ControlPassword is a bad indicator. Having a remote Tor server configured is a way better indicator.